### PR TITLE
fix(getting-started): Removed extra links

### PR DESCRIPTION
**Pull request by dlabrecq**
- Removed extra home and profile links.
- Removed setTimeout code to route to home -- no longer needed.

### CODE REVIEW

File: src/app/getting-started/getting-started.component.html [hunk 1]
```diff
@@ -120,9 +120,8 @@ <h2 class="circle">2</h2>
           <div class="success-border">
             <h2 class="success-title">Success! You're ready to start in OpenShift.io.</h2>
             <p>
-              Go to your <a [routerLink]="['/', '_home']">home</a> page to start coding, plan work, or do other things.
-              You can also fill out more information about you in your
-              <a [routerLink]="['/', '_home']">profile</a> so that teammates can identify you.
+              Go to your home page to start coding, plan work, or do other things.
+              You can also fill out more information about you in your profile so that teammates can identify you.
```
Comment (joshuawilson) [inline src/app/getting-started/getting-started.component.html]: I'm not sure this is the text we want. @catrobson can you please review?

Comment (catrobson) [inline src/app/getting-started/getting-started.component.html]: New text now that we had to remove the links should be:
"You can now proceed to your home page to plan work, code, and deploy your applications."

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6f6c44ed01478ce2f9fb31b01c079cf159836ba1
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@dlabrecq snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1154-fabric8-ui.int.rdu2c.fabric8.io

**Pull request merged**